### PR TITLE
katib-6516

### PR DESCRIPTION
**Pull request by pallavipannu**

### CODE REVIEW

File: mnist/README.md [hunk 1]
```diff
@@ -34,11 +34,21 @@
...
 
+## Tuning.yaml file Details:
+1. **objective**: The metric that you want to optimize. 
+2. **goal** parameter is mandatory in tuning.yaml file.
+3. **objectiveMetricName:** Katib uses the objectiveMetricName and additionalMetricNames to monitor how the hyperparameters work with the model. Katib records the value of the best objectiveMetricName metric.
+4. **parameters** : The range of the hyperparameters or other parameters that you want to tune for your machine learning (ML) model.
+5. **parallelTrialCount**: The maximum number of hyperparameter sets that Katib should train in parallel. The default value is 3.
+6. **maxTrialCount**: The maximum number of trials to run.
+7. **maxFailedTrialCount**: The maximum number of failed trials before Katib should stop the experiment.
+8. **algorithm**: The search algorithm that you want Katib to use to find the best hyperparameters or neural architecture configuration.
```
Comment (deepio-oc) [inline mnist/README.md:50]: can we list all possible values for algorithm ?

Comment (pallavipannu) [reply mnist/README.md:50]: Done @deepio-oc , Listed all the possibe values. we can merge the pr now.

File: mnist/README.md [hunk 2]
```diff
@@ -47,7 +47,7 @@
 5. **parallelTrialCount**: The maximum number of hyperparameter sets that Katib should train in parallel. The default value is 3.
 6. **maxTrialCount**: The maximum number of trials to run.
 7. **maxFailedTrialCount**: The maximum number of failed trials before Katib should stop the experiment.
-8. **algorithm**: The search algorithm that you want Katib to use to find the best hyperparameters or neural architecture configuration. 
+8. **algorithm**: The search algorithm that you want Katib to use to find the best hyperparameters or neural architecture configuration. Search algorithms   available in Katib are Grid search, Random search, Bayesian optimization, Hyperband, Tree of Parzen Estimators (TPE), Covariance Matrix Adaptation Evolution Strategy (CMA-ES), Neural Architecture Search based on ENAS, Differentiable Architecture Search (DARTS).
```
Comment (deepio-oc) [inline mnist/README.md]: We need to give the exact value which user needs to enter in this field. If user enters 'Adaptation Evolution Strategy (CMA-ES)', that value won't work right ?

You can say following:
Search algorithm to find the best hyper parameters. Value must be one of following:
1.
2.
3.

Comment (pallavipannu) [reply mnist/README.md]: Done @deepio-oc . Currently darts is in alpha with limited support, we will not be able to see the results in kubeflow ui n case of darts, rest are added.

**Pull request merged**